hpOneView/resources/facilities/power_devices.py

```diff
@@ -47,14 +47,227 @@ def __init__(self, con):
         self._connection = con
         self._client = ResourceClient(con, self.URI)
 
-    def get_utilization(self, id, fields=None, filter=None, refresh=False, view=None):
+    def get_all(self, start=0, count=-1, filter='', query='', sort=''):
+        """
+        Gets a set of power delivery device resources according to the specified parameters. Filters can be used to get
+        a specific set of power delivery devices. With no filters specified, the API returns a potentially paginated
+        list of all the power delivery device resources subject to start/count/sort parameters.
+
+        Args:
+            start:
+                The first item to return, using 0-based indexing.
+                If not specified, the default is 0 - start with the first available item.
+            count:
+                The number of resources to return. A count of -1 requests all the items.
+                The actual number of items in the response may differ from the requested
+                count if the sum of start and count exceed the total number of items, or
```

Comment:
", or if returning the requested number of items would take too long." is outdated. Currently the resource client handles that situation.